core/server/services/routing/helpers/fetch-data.js

```diff
@@ -48,7 +48,7 @@ function processQuery(query, slugParam, locals) {
     });
 
     // Return a promise for the api query
-    return api[query.resource][query.type](query.options);
+    return (api[query.alias] || api[query.resource])[query.type](query.options);
```

Comment:
Would be cool to just do `api[query.controller]`.
The reason why there is a differentiation is that v0.1 uses different controllers than v2.
pages vs posts
users vs authors

Fix me